packages/element/src/test/serialize.js

```diff
@@ -303,26 +303,22 @@ describe( 'renderNativeComponent()', () => {
 } );
 
 describe( 'renderComponent()', () => {
-	it( 'calls constructor and componentWillMount', () => {
+	it( 'calls constructor', () => {
 		class Example extends Component {
 			constructor() {
 				super( ...arguments );
 
 				this.constructed = 'constructed';
 			}
 
-			componentWillMount() {
```

Comment:
@youknowriad, let's keep this test unchanged and remove it together with deprecation as part of 3.3 release.

I think you can merge it after this change is reverted. I guess you will have to update the test to catch the warning.